socorro/unittest/external/es/test_supersearch.py

```diff
@@ -1126,6 +1126,273 @@ def dt_to_midnight(date):
         )
 
     @minimum_es_version('1.0')
+    def test_get_with_number_histogram(self):
+        yesterday = self.now - datetime.timedelta(days=1)
+        the_day_before = self.now - datetime.timedelta(days=2)
+
+        time_str = '%Y%m%d%H%M%S'
+        today_int = int(self.now.strftime(time_str))
+        yesterday_int = int(yesterday.strftime(time_str))
+        day_before_int = int(the_day_before.strftime(time_str))
+
+        self.index_crash({
+            'signature': 'js::break_your_browser',
+            'product': 'WaterWolf',
+            'os_name': 'Windows NT',
+            'build': today_int,
```

Comment:
Nope, build_ids are integers, and stored as integers in Elasticsearch. Which is nice because thus we can use int operators on them, like ``<`` or ``>``. :)